### DISCUSSION TITLE
Crashlytics not uploading symbols

### DISCUSSION THREAD

**Discussion by Shresht20** [firebase/firebase-ios-sdk#8224] [Q&A] [answered]
Hello there,
I am an iOS developer, right now I am facing a problem in uploading symbols to Crashlytics. The problem is, my app is not automatically uploading symbols. I have followed all the instructions from the Firebase docs. I really appreciate those, who help me solve this issue.
Thanks

**Comment by samedson** [comment, marked as answer]
Hey @Shresht20, can you show me what your Build Phase that runs Crashlytics looks like? In addition, I want to make sure you've set the "Debug Information Formation" Build Setting to "Dwarf with DSYM".

How are you integrating the SDK? Any errors you're getting?

**Reply by Shresht20** [reply]
Hi @samedson , 
thanks for replying to my message.

1. Here's a picture of my Build Phase that runs Crashlytics, please have a look.
![Screenshot 2021-06-11 at 22 35 26](https://user-images.githubusercontent.com/81809969/121724328-54a05200-cb05-11eb-95f9-b867e7083e80.png)

2. Here's a picture of my Build Settings with "Debug Infromation Format"- "Dwarf with DSYM file"
![Screenshot 2021-06-11 at 22 39 46](https://user-images.githubusercontent.com/81809969/121724747-ee67ff00-cb05-11eb-9c77-b1a001d0e8c7.png)

3. I used Swift Package Manager to integrate Firebase SDKs in my project.

**Reply by samedson** [reply]
So the first line `${PODS_ROOT}/FirebaseCrashlytics/run` is probably failing because the run script is in a different directory (since you're using SPM). I would suggest removing all lines except line 5 of your run script and trying that.

**Reply by Shresht20** [reply]
Thanks for that, I will try and let you know the result.

**Reply by Shresht20** [reply]
Thanks, it worked! Another question, if you don't mind, is Firebase Performance available for SPM?

**Reply by paulb777** [reply]
Not yet. We're working on it and you can follow progress at #6528

**Reply by Shresht20** [reply]
Thanks @paulb777 !
